entity-management-web/src/main/java/eu/europeana/entitymanagement/EntityManagementApp.java

```diff
@@ -1,24 +1,31 @@
 package eu.europeana.entitymanagement;
 
+import eu.europeana.entitymanagement.common.config.SocksProxyConfig;
+import eu.europeana.entitymanagement.config.SocksProxyActivator;
 import org.apache.logging.log4j.LogManager;
+import org.springframework.batch.core.configuration.annotation.EnableBatchProcessing;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.actuate.autoconfigure.security.servlet.ManagementWebSecurityAutoConfiguration;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
 import org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration;
 import org.springframework.boot.builder.SpringApplicationBuilder;
 import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
 
-import eu.europeana.entitymanagement.common.config.SocksProxyConfig;
-import eu.europeana.entitymanagement.config.SocksProxyActivator;
-
 /**
  * Main application. Allows deploying as a war and logs instance data when deployed in Cloud Foundry
  */
 //@SpringBootApplication(scanBasePackages = {"eu.europeana.entitymanagement", "eu.europeana.api.commons"},
 @SpringBootApplication(scanBasePackages = {"eu.europeana.entitymanagement"},
-        // Remove these exclusions to re-enable security
-        exclude = { SecurityAutoConfiguration.class, ManagementWebSecurityAutoConfiguration.class }
-        )
+        exclude = {
```

Comment:
do we need these exclusions given the scanBasePackages?

Reply:
These are [AutoConfiguration classes](https://docs.spring.io/spring-boot/docs/1.3.3.RELEASE/reference/html/using-boot-auto-configuration.html).

They are always loaded by default, due to the  `@SpringBootApplication` annotation.